pkg/tsdb/azuremonitor/credentials_test.go

```diff
@@ -170,26 +170,22 @@ func TestCredentials_getAzureCredentials(t *testing.T) {
 	})
 
 	t.Run("when auth type is client secret", func(t *testing.T) {
-		jsonData := simplejson.NewFromAny(map[string]interface{}{
-			"azureAuthType": azcredentials.AzureAuthClientSecret,
-			"cloudName":     azUSGovManagement,
-			"tenantId":      "9b9d90ee-a5cc-49c2-b97e-0d1b0f086b5c",
-			"clientId":      "849ccbb0-92eb-4226-b228-ef391abd8fe6",
-		})
+		jsonData := &types.AzureClientSettings{
+			AzureAuthType: azcredentials.AzureAuthClientSecret,
+			CloudName:     azureMonitorUSGovernment,
+			TenantId:      "9b9d90ee-a5cc-49c2-b97e-0d1b0f086b5c",
+			ClientId:      "849ccbb0-92eb-4226-b228-ef391abd8fe6",
+		}
 
 		t.Run("should return client secret credentials", func(t *testing.T) {
-			cfg := &setting.Cfg{
-				Azure: &azsettings.AzureSettings{
-					Cloud: azsettings.AzureChina,
-				},
-			}
+			cfg := &setting.Cfg{}
 
 			credentials, err := getAzureCredentials(cfg, jsonData, secureJsonData)
 			require.NoError(t, err)
 			require.IsType(t, &azcredentials.AzureClientSecretCredentials{}, credentials)
 			clientSecretCredentials := credentials.(*azcredentials.AzureClientSecretCredentials)
 
-			assert.Equal(t, azsettings.AzureChina, clientSecretCredentials.AzureCloud)
+			assert.Equal(t, azsettings.AzureUSGovernment, clientSecretCredentials.AzureCloud)
```

Comment:
Why the change in the configuration here?

Reply:
In previous version we set up test like this 
```
	jsonData := simplejson.NewFromAny(map[string]interface{}{
			"azureAuthType": azcredentials.AzureAuthClientSecret,
			"cloudName":     azUSGovManagement,
			"tenantId":      "9b9d90ee-a5cc-49c2-b97e-0d1b0f086b5c",
			"clientId":      "849ccbb0-92eb-4226-b228-ef391abd8fe6",
		})
```
`azUSGovManagement` is not a string. It is route defined in `routes.go`. Then in test we add following configuration 
```
			cfg := &setting.Cfg{
				Azure: &azsettings.AzureSettings{
					Cloud: azsettings.AzureChina,
				},
			}
```
And because `azUSGovManagement` is not a string we took cloud name from `cfg.Azure.Cloud` 
So i decide to  change config that cloudName in test setup will be equal in assertion. Which is  AzureUSGovernment